src/app/features/characters/characters.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="wrapper">
+
+  <app-random-character></app-random-character>
+
+  <div div class="wrapper-all-chars">
+    <div class="wrapper-characters">
+      <app-character-card
+        *ngFor="let char of list$ | async"
+        [char]="char"
+        (idToShow)="showCharacter($event)"
+      >
+      </app-character-card>
+    </div>
+
+    <div class="wrapper-aside">
+      <app-character-info
+        *ngIf="char; else pleaseSelect"
```

Comment:
_char_ is a bad name for boolean variable, isCharSelected or similar would be better

Reply:
thanks, changes will be in the next PR